pkg/codegen/docs/templates/constructor_args.tmpl

```diff
@@ -1,19 +1,15 @@
 {{ define "constructor_args" }}
 <dl class="resources-properties">
-    <dt class="property-required" title="Required">
-        <span>name</span>
+  {{ range $params := . }}
+    <dt class="property-xxx" title="xxx">
```

Comment:
Is that class name legit? And that `title` attribute value too.

Reply:
It is not! Let me fix that 🙃

Reply:
Updated the PR to now mark params as optional/required, e.g.

![image](https://user-images.githubusercontent.com/4029847/79176131-56ad3a80-7db4-11ea-80fd-9fa28efa2680.png)